frontend/javascripts/oxalis/view/action-bar/merge_modal_view.tsx

```diff
@@ -267,17 +226,28 @@ class _MergeModalView extends PureComponent<Props, MergeModalViewState> {
                 }))}
               />
             </Form.Item>
-
-            <ButtonWithCheckbox
-              checkboxContent={mergeIntoActiveTracingCheckbox}
-              button={
-                // @ts-expect-error ts-migrate(2322) FIXME: Type '"default"' is not assignable to type 'SizeTy... Remove this comment to see the full error message
-                <Button type="primary" size="default" disabled={this.state.selectedProject == null}>
+            <Form.Item>
+              <Tooltip title="Imports trees and tree groups directly into the currently opened annotation.">
```

Comment:
```suggestion
              <Tooltip title="Imports trees and tree groups (no volume data!) directly into the currently opened annotation.">
```

or is this too much?